### ISSUE TITLE
Second check?

### ISSUE THREAD

**Issue by mirh** [UCyborg/LegacyD3DResolutionHack#2]
https://sourceforge.net/p/dxwnd/discussion/general/thread/bee67290/#c6c7/2c59/2f39

**Comment by UCyborg** [comment]
I did notice the second occurrence, I just didn't touch it because nobody knows for sure what's it for. It would be nice if someone could confirm if it really is related to screen resolution.

**Comment by mirh** [comment]
@ghotik

**Comment by fluxx0r** [comment]
I installed the newest version of thirteens widescreenfix and I used the d3dresolutionhack, now my game starts with 21:9 format in 2560x1080 but when I drive a very fast car I have graphic problems, maybe anyone knows a solution?
https://www.youtube.com/watch?v=JeKS1T5Smmo
I'm using windows 10, with a nividia 980gtx ti, newest updates and drivers are installed

**Comment by fluxx0r** [comment]
problem seems to be solved. I had to change to zoomfactor from the widescreenfix from auto to 4.0 now it works!

**Comment by elishacloud** [comment]
The second occurrence is located inside the DLLMain entry point function (with both d3dim.dll and d3dim700.dll).  I could not tell for sure what it was checking (since I don't really want to spend the time to debug an entry point function) but I don't think it has to do with the resolution.

However, either way the code only seems to run when the dll is loading so changing it would not do any good since LegacyD3DResolutionHack is only able to change the code after the dll is already loaded.  At that point it is too late since the code has already executed.

Bottom line: I think it is safe to ignore the second check.
